Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Scanner/QRScanningViewModel.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public protocol QRScanningViewModel {
+    var title: String { get }
+    var instructionText: String { get }
+    var successMessage: String { get }
+    var errorMessage: String { get }
+    var shouldShowAlert: Bool { get }
+    var shouldDismissViewAfterScanComplete: Bool { get }
+    var format: String? { get }
+    var alertTitle: String { get }
+    var alertMessage: String { get }
+    var alertAction: String { get }
+}
```

Comment:
This has been removed scanner will now just pass the resulting value from the scan to the view model, it will be down the concrete implementation of view model to handle validation in the client app/framework